src/components/organisms/AdminDashboard/AdminDashboard.tsx

```diff
@@ -72,7 +76,7 @@ export const AdminDashboard: React.FC = () => {
 
   const hasVenues = renderedPartyVenues.length > 0;
 
-  if (isLoading) {
+  if (isLoading || !isLoaded) {
```

Comment:
Maybe rename to distinguish:
```suggestion
  if (isLoadingVenues || !isLoadedWorld) {
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2474/commits/810959fc143f367fa1c4c139201437a894fc3b54